content/strategy-goals/strategy/working-with-use-cases.md

```diff
@@ -25,12 +31,39 @@ For Sales & CE, a good use case is:
 - Demonstrable; tell a cohesive narrative by tying together all parts of our product
 - Able to be tested discretely and completely (timeboxed) by a customer to prove out the value
 
-#### Product & Engineering
+# Applied in Product & Engineering
+
+This section provides context on how use cases align to our platform, products, features and roadmap to set forth a framework teams can leverage to make use cases first class citizens in our platform.
+
+The use cases are problems that our customers encounter that our product helps solve. In Product and Engineering, we’re building for these scenarios, demonstrating a clear progression of solving our customers (and potential customers) pain points over time. When we talk about what’s coming next on the roadmap, we think about how we’re enhancing the product to better solve these use cases. They provide general alignment and focus that the team is constantly thinking about, across the organization and within our product teams and features. When we package together Sourcegraph’s features like search, code insights, code intelligence, batch changes, notebooks - we’re able to create a cohesive and compelling narrative and solution in a particular use case for the end user.
+
+## How do use cases fit within our platform?
+
+Companies have different workflows for each use case, for example how they monitor and fix security vulnerabilities. In turn, our platform provides products and features that collectively create solutions for each use case. Our customers leverage Sourcegraph in innovative ways to solve for use cases they care about, that's how we deliver value. This is how it fits together:
+
+- Platform : the overall Sourcegraph offering
+- Product : one of the vertical products in our offering (e.g. search, code monitor, code insights, batch changes)
+- Feature : a capability within a products
+- Solution : any collection of products and features that solve a problem or use case
+
+If a customer cares about Code Health, they may use Code Monitoring differently than one that cares about monitoring for Security Vulnerabilities. Our focus is to enrich our offering in each one of these products to support solving for each use case. This will make it easiery for our users to recognize where Sourcegraph could provide value in their workflows.
```

Comment:
```suggestion
If a customer cares about Code Health, they may use Code Monitoring differently than one that cares about monitoring for Security Vulnerabilities. Our focus is to enrich our offering in each one of these products to support solving for each use case. This will make it easier for our users to recognize where Sourcegraph could provide value in their workflows.
```